argus/client/windows.py

```diff
@@ -67,6 +68,7 @@ def __init__(self, hostname, username, password,
         self._password = password
         self._cert_pem = cert_pem
         self._cert_key = cert_key
+        self.manager = get_windows_action_manager(self)
```

Comment:
It should be use outside of the object when for example a tests needs to download something they should use `download` method.